lvsfunc/deblock.py

```diff
@@ -142,11 +149,14 @@ def _eval_db(n: int, f: Sequence[vs.VideoFrame],
     return kernel.resample(debl, format=clip.format, matrix=targ_matrix if not is_rgb else None)
 
 
-def vsdpir(clip: vs.VideoNode, strength: VSDPIR_STRENGTH_TYPE = 25, mode: str = 'deblock',
-           matrix: Matrix | int | None = None, tiles: int | Tuple[int] | None = None,
-           cuda: bool | Literal['trt'] = True, i444: bool = False, kernel: Kernel | str = Catrom(),
-           zones: List[Tuple[Range | List[Range] | None, VSDPIR_STRENGTH_TYPE]] | None = None,
-           **dpir_args: Any) -> vs.VideoNode:
+def vsdpir(
+    clip: vs.VideoNode, strength: VSDPIR_STRENGTH_TYPE = 25, mode: str = 'deblock',
```

Comment:
Gonna take the opportunity to slightly improve upon some stuff that's annoyed me for a while. Could you set the default strength to 10?